### PR TITLE
Remove beak from CommandBar's menu in Large+ screen sizes

### LINKED ISSUE #1988: Remove beak on command bar menus for L and > breakpoints
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) No

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable) OD, Outlook, SharePoint

#### Describe the issue:

We updated the design guidance for beaked menus. Beaks should only appear on S-M breakpoints (320-639px).

#### Actual behavior:
Beaks appear on all menus across all breakpoints

#### Expected behavior:
Only show beaks on L and > breakpoints (640px and [>)]([url](url))

### PR DESCRIPTION
**Pull request by atneik**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #1988
- [x] Include a change request file using `$ npm run change`

#### Description of changes

As per Design, remove beak from CommandBar's and BaseButton's contextual menu

#### Focus areas to test

https://odsp-int.azurewebsites.net/anihan/remove-beak/#/examples/commandbar

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx [hunk 1]
```diff
@@ -143,7 +146,7 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
         { (contextualMenuProps) ?
           (<ContextualMenu
             className={ css('ms-CommandBar-menuHost') }
-            isBeakVisible={ true }
+            isBeakVisible={ isScreenSmall ? true : false }
```
Comment (christiango) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx]: Instead of a ternary operator, just use the value of isScreenSmall

**Comment by christiango** [comment]
Seems like from the code this is just for small screen sizes. Can you update the description of the PR to reflect that? Also, why is it only for small screen sizes?

**Comment by dzearing** [comment]
No screenshots?

**Comment by atneik** [comment]
@christiango, @angallison will be better answer that question for you

@dzearing are screenshots needed in case I have a dev link?

File: packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx [hunk 2]
```diff
@@ -24,6 +25,7 @@ export interface IBaseButtonState {
   menuProps?: IContextualMenuProps | null;
 }
 
+@withResponsiveMode
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx]: I'd really prefer not to do this for every button. 

A better solution might be to have a "default" beakVisibility logic in contextual menu. Forcing everyone to consider this at the consumer layer results in inconsistency.

**Pull request merged**